app/dashboard/templates/profile_details.html

```diff
@@ -196,81 +194,91 @@
                       </p>
                     </div>
-                  {% endif %}
+                    {% endif %}
+                    <div class="tag token">
+                      <p>
+                        {{ tip.amount }}
+                        <span>{{ tip.tokenName }}</span>
+                      </p>
+                    </div>
+                    {% if tip.value_in_usdt_now %}
+                      <div class="tag usd">
+                        <p>
+                          {{ tip.value_in_usdt_now }}
+                          <span>USD</span>
+                        </p>
+                      </div>
+                    {% endif %}
+                  </div>
                 </div>
-              </div>
-            {% endfor %}
+              {% endfor %}
             {% else %}
-            {% trans "No tips" %}
+              {% trans "No tips" %}
             {% endif %}
-
-            <button type="button" class="profile-bounties__btn-show-all btn btn-outline-secondary btn-block mt-1 py-2 hidden">{% trans "Show all" %}</button>
+            <div class="col-12 text-center">
+              <button type="button" class="profile-bounties__btn-show-all btn btn-outline-secondary mt-1 py-2 hidden">{% trans "Show all" %}</button>
+            </div>
           </div>
         </div>
       {% endif %}
 
       {% if activities %}
         <div class="row">
-          <div class="col py-4">{% trans "Latest Activities" %}</div>
+          <div class="col pt-4 py-2">{% trans "Latest Activities" %}</div>
         </div>
         <div class="row">
           <div class="container profile-bounties profile-bounties--activities">
             {% for group in activities %}
               <div class="row">
                 <div class="col-12">
-                  <p><small>{{ group.title }}</small></p>
+                  <span class="font-caption profile-bounties__sort_label">{{ group.title }}</span>
                 </div>
                 {% if group.completed %}
                   <div class="container">
-                    <div class="row profile-bounties__activities-group mb-3 p-2">
+                    <div class="row mb-3 p-2">
                       <div class="col-12">
-                        <div class="profile-bounties__group-title profile-bounties__group-title--completed">{% trans "Completed" %} {{ group.completed|length }} {% if group.completed|length > 1 %}{% trans "Bounties" %}{% else %}{% trans "Bounty" %}{% endif %} <img src="{% static "v2/images/check-green.svg" %}"></div>
-
+                        <div class="profile-bounties__group-title profile-bounties__group-title--completed">
+                          <i class="far fa-check-circle"></i>
+                          <span class="pl-1">
+                            {% trans "Completed" %} {{ group.completed|length }}
+                            {% if group.completed|length > 1 %}
+                              {% trans "Bounties" %}
```

Comment:
We could use `pluralize` here:

```html
{% blocktrans %}
Bount{{ group.submitted|pluralize:"y,ies" }}
{% endblocktrans %}
```

More info: https://docs.djangoproject.com/en/2.0/ref/templates/builtins/#pluralize

Reply:
Did not know that ! Thanks :D

Reply:
Shouldn't need that. 🤔